docs/adr/0029-formater-template-ember.md

```diff
@@ -0,0 +1,104 @@
+Les linter proposent des configurations standard, aussi appelés `preset`, qui rassemblent des règles, par exemple celle 
+de [eslint](https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js). 
+
+#### Formater
+
+Certains formaters ne proposent quasiment aucune configuration, et ne garantissent pas la continuité des 
+règles appliquées, c'est à dire qu'une nouvelle release du formater peut entraîner un nouveau formatage
+en l'absence de modification du fichier.
+
+[L'argumentation](https://prettier.io/docs/en/why-prettier.html) d'un de ces formater, `Prettier` est que cela
+permet au développeur de se concentrer sur d'autres tâches lors de l'implémentation et de la revue.
+
+Par exemple, il existe [20 options pour Prettier](https://prettier.io/docs/en/options.html#end-of-line))
+
+Un exemple de comportement non configurable est la présence obligatoire d'une ligne vide en fin de fichier,
+[spécifié par POSIX](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795).
+La plupart du temps, le code est exécuté sur des distributions Linux non certifiées POSIX: faut-il faire figurer
+cette ligne vide ? Prettier répond ["Oui" partout](https://github.com/prettier/prettier/issues/6360).
+
+Pour résumer
+> By far the biggest reason for adopting Prettier is to stop all the ongoing debates over styles.
+[Source](https://prettier.io/docs/en/option-philosophy.html)
+
+### Templates Ember (hbs)
+
+Les templates utilisés par Ember (hbs - handlebars) ont pour cible le html.
+
+Un html peut présenter :
+- une structure assez profonde (nombre de niveaux), malgré l'approche orientée composants;
+- des éléments avec de nombreux attributs.
+
+De plus, il n'existe pas d'éditeur visuel handlebars. 
+
+En conséquence, un formatage approprié est important. 
+
+### Approche Pix
+Jusqu'ici, l'approche utilisée est:
+- outil de lint;
+- avec de nombreuses [configurations spécifiques](../../.eslintrc.yaml).
+
+Nous ne connaissons pas le besoin à l'origine des configurations spécifiques.
+
+
+### Solution n°1: utiliser les règles syntaxiques du linter natif
+Le linter natif [ember-template-lint](https://github.com/ember-template-lint) propose des règles d'inspection du 
+formatage. Elles sont embarquées dans le preset `stylistic`, voici une [règle en exemple](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/linebreak-style.md).
+
+Avantages:
+- configurable (est-ce un besoin ?)
+- bibliothèque déjà présente
+
+Inconvénients:
+- pas d'option `auto-fix` sur le formatage [pour l'instant](https://github.com/ember-template-lint/ember-template-lint/issues/1180): la reprise d'historique (ex: indentation) doit notamment être faite manuellement
+
+[Exemple d'application](https://github.com/1024pix/pix/tree/tech-format-admin-hbs-eslint) sur pix-admin
+
+### Solution n°2: utiliser le formater Prettier
+
+Avantages:
+- non configurable (est-ce un besoin ?)
```

Comment:
really, i thought it was.. tu ne peux pas creer un .prettierrc?

Reply:
C'est possible oui

Reply:
Certes, mais il n'y a que 20 options.. autant dire presque aucune (voir ADR)

Reply:
J'ai précisé le nombre dans l'ADR